projects/netgrif-components/src/lib/data-fields/data-field.theme.scss

```diff
@@ -228,6 +228,27 @@
     .form-input-error {
         border-color: mat.get-color-from-palette($warn) !important;
     }
+
+    .svg-inactive-stroke {
+        stroke: black !important;
+    }
+
+    .svg-inactive-fill {
+        fill: black !important;
+    }
+
+    .svg-active-stroke {
+        stroke: map-get($primary, A200) !important;
+    }
+
+    .svg-active-fill {
+        fill: map-get($primary, A200) !important;
+    }
+
+    .svg-transition-enabled {
+        stroke: green;
+        fill: yellowgreen;
```

Comment:
palette colors?

Reply:
if you want another colours in project, you can define that class by your own in project